docs/excel/custom-functions-overview.md

```diff
@@ -163,7 +163,7 @@ function getTemperature(thermometerID){
 
 ## Streamed functions
 
-Streamed custom functions let you output data to cells repeatedly over time, without waiting for Excel or users to request recalculations. For example, as shown in the following code, the custom function incrementValue adds a number to the result every second, and Excel displays each new value automatically using the setResult callback. To see the registration code used with incrementValue, see the customfunctions.js file.
+Streamed custom functions let you output data to cells repeatedly over time, without waiting for Excel or users to request recalculations. For example, the `incrementValue` custom function in the following example adds a number to the result every second, and Excel displays each new value automatically using the `setResult` callback. To see the registration code used with `incrementValue`, read the *customfunctions.js* file.
```

Comment:
@saunders77 can you change "...in the following example..." to "...in the following code..." because the word example is used twice in the same sentence. Thanks!